src/core/Note.dart

```diff
@@ -0,0 +1,128 @@
+import 'note_app.dart';
+class Note {
+  
+   String? Title, Content;
```

Comment:
consider using non-nullable types for your class properties if possible. This can help catch potential null-related errors during development.

String title;
String content;

Note.create({required this.title, required this.content});